### PR TITLE
Include option to restrict bucket downloads to same region (SC-240)

### PR DESCRIPTION
**Pull request by zimingd**
depends on https://github.com/Sage-Bionetworks/service-catalog-library/pull/241 and #84

### CODE REVIEW

File: sceptre/scipool/templates/sc-product-s3-private-enc.yaml [hunk 1]
```diff
@@ -53,6 +53,10 @@ Resources:
           Info:
             LoadTemplateFromURL: !Sub '${RepoRootURL}/v1.1.10/s3/sc-s3-encrypted-ra.yaml'
           Name: 'v1.1.10'
+        - Description: 'Include option to restrict bucket downloads to same region'
+          Info:
+            LoadTemplateFromURL: !Sub '${RepoRootURL}/1.1.15/s3/sc-s3-encrypted-ra.yaml'
+          Name: 'v1.1.15'
```
Comment (zaro0508) [inline sceptre/scipool/templates/sc-product-s3-private-enc.yaml:59]: next version is `v1.1.11`

File: sceptre/synapsedev/templates/deployer-role.json [hunk 2]
```diff
@@ -1,63 +1,63 @@
-{
```
Comment (zimingd) [inline sceptre/synapsedev/templates/deployer-role.json:1]: This file had mixed line endings.

**Review by zaro0508 — CHANGES_REQUESTED** [review]
i think you committed extra stuff?

File: sceptre/synapsedev/templates/SynapseCMK-template.json [hunk 3]
```diff
@@ -1,341 +1,341 @@
-{
```
Comment (zimingd) [inline sceptre/synapsedev/templates/SynapseCMK-template.json:1]: This file had mixed line endings. No semantic change

File: sceptre/scipool/templates/sc-product-s3-private-enc.yaml (hunk 1 above)
Comment (zimingd) [reply sceptre/scipool/templates/sc-product-s3-private-enc.yaml:59]: We've gone up to `v1.1.14` according to the tags
https://github.com/Sage-Bionetworks/service-catalog-library/tags

Comment (zaro0508) [reply sceptre/scipool/templates/sc-product-s3-private-enc.yaml:59]: ahh ok. lets do `v.1.1.15` then

File: sceptre/scipool/templates/sc-product-s3-private-enc.yaml [hunk 4]
```diff
@@ -53,6 +53,10 @@ Resources:
           Info:
             LoadTemplateFromURL: !Sub '${RepoRootURL}/v1.1.10/s3/sc-s3-encrypted-ra.yaml'
           Name: 'v1.1.10'
+        - Description: 'Include option to restrict bucket downloads to same region'
```
Comment (zaro0508) [inline sceptre/scipool/templates/sc-product-s3-private-enc.yaml:56]: `Last updated at ${StackDatetime}` on v1.1.10 should only be on the latest version.  Would you mind removing that on v1.1.10 and addeding it to the end of this description?

File: sceptre/scipool/templates/sc-product-s3-synapse.yaml [hunk 5]
```diff
@@ -61,6 +61,10 @@ Resources:
           Info:
             LoadTemplateFromURL: !Sub '${RepoRootURL}/v1.1.10/s3/sc-s3-synapse-ra.yaml'
           Name: 'v1.1.10'
+        - Description: 'Include option to restrict bucket downloads to same region'
```
Comment (zaro0508) [inline sceptre/scipool/templates/sc-product-s3-synapse.yaml:64]: same here please

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Pull request merged**